features-json/push-api.json

```diff
@@ -308,7 +308,7 @@
       "15.4":"n #3",
       "15.5":"n #3",
       "15.6":"n #3",
-      "16.0":"a #6",
```

Comment:
This is correct. Safari 16 for macOS Ventura will be out in October, numbered 16.1.

Reply:
Ah, I should have marked line 312 for this comment... but you get the idea.

Reply:
I do, thank you! :)